src/components/About/About.module.css

```diff
@@ -1,28 +1,186 @@
 .wrap {
-  text-align: left;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  text-align: center;
   font-family: 'Courier New', Courier, monospace;
+  margin: auto;
+  padding: 20px;
 }
 
 .preload {
-  position: relative;
-  left: 4em;
-  top: 2em;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 500px;
 }
 
 .title {
   text-align: center;
   font-weight: bold;
   margin-bottom: 20px;
+  font-family: 'Courier New', Courier, monospace;
+  font-size: 35px;
 }
 
 .repo {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 500px;
   font-size: 25px;
   font-weight: 300;
-  margin: 0 auto;
 }
 
-.repolist {
+.info {
+  display: flex;
+  align-items: center;
+  font-size: 30px;
+}
+
+.bio {
+  display: flex;
+  flex-direction: column;
+  font-size: 15px;
+}
+
+.name {
+  text-decoration: none;
+  font-size: 25px;
+  color: black;
+}
+
+.name:hover,
+.link:hover {
+  opacity: 0.5;
+}
+
+.image {
+  width: 220px;
+  height: auto;
+
+}
+
+.photo {
+  width: 75%;
+  height: 75%;
+  border-radius: 80%;
+}
+
+.repoList {
+  list-style-type: none;
+  padding: 10px;
+  width: 400px;
+}
+
+.link {
```

Comment:
Codacy found an issue: [Expected selector ".link" to come before selector ".link:hover" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6313319)